app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +28,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <receiver android:name=".common.data.notification_service.NotificationBroadcastReceiver" />
```

Comment:
It's also important to listen to when the Android device boots up, since all AlarmManager reminders will be cleared when the device is turned off, so you'll have to reschedule